plugins/services/src/js/utils/MarathonErrorUtil.js

```diff
@@ -121,6 +121,15 @@ const MarathonErrorUtil = {
         });
       }
 
+      if (!Array.isArray(errors) || errors.length === 0) {
```

Comment:
We also could populate errors and have the following reduce do the work, why do you think this is a better solution?